datumaro/components/exporter.py

```diff
@@ -279,3 +279,122 @@ def _save_point_cloud(self, item=None, path=None, *, name=None, subdir=None, bas
 
     def _save_meta_file(self, path):
         save_meta_file(path, self._extractor.categories())
+
+
+# TODO: Currently, ExportContextComponent is introduced only for Datumaro and DatumaroBinary format for multi-processing.
+# We need to propagate this to everywhere in Datumaro 1.2.0
+class ExportContextComponent:
```

Comment:
Would `Exporter` be replaced by `ExportercontextComponent` in 1.2.0?

Reply:
No `Exporter` will have `ExporterContextComponent`. It's the component pattern.